### PR TITLE
PAYEE-964: Fix overlapping text & Provide unique ids

### PR DESCRIPTION
**Pull request by PerrettJ4**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/incomeTaxSummaryIncomeSourceDetail.scala.html [hunk 1]
```diff
@@ -54,7 +54,7 @@ <h3 id="@{typeLabel}@{idx}Heading" class="govuk-heading-l heading-large flush br
                         @for((taxCodeList, listIdx) <-
                         taxCodeViewModel.get(incomeSource.taxCode).map(_.taxCodeDetails).getOrElse(Seq.empty[DescriptionListViewModel])
                         zip (Stream from 1)) {
-            <div  style="display: none;" id="taxCodeList@{listIdx.toString}">
+            <div  style="display: block;" id="@{typeLabel}TaxCodeList@{pidx.toString}">
```
Comment (masahmad) [inline app/views/incomeTaxSummaryIncomeSourceDetail.scala.html]: shouldn't it be hidden ie none?

File: test/views/html/IncomeTaxSummaryViewSpec.scala [hunk 2]
```diff
@@ -40,8 +40,10 @@ class IncomeTaxSummaryViewSpec extends TaiViewSpec {
       val view = template(vm, viewModel, appConfig)
       val doc = Jsoup.parse(view.toString())
 
-      doc must haveElementAtPathWithText("#taxCodeTerm_1_1", messages("tai.taxCode.part.announce", "K") + " K")
-      doc must haveElementWithId("taxCodeDescription_1_1")
+      doc must haveElementAtPathWithText(
+        "#employmentTaxCodeTerm_1_1",
+        messages("tai.taxCode.part.announce", "K") + " K")
```
Comment (sergi-hmrc) [inline test/views/html/IncomeTaxSummaryViewSpec.scala:45]: u expecting K twice?

Comment (PerrettJ4) [reply test/views/html/IncomeTaxSummaryViewSpec.scala:45]: I believe the first only locates the element, and the second is the _K_ that it is checking for

Comment (PerrettJ4) [reply test/views/html/IncomeTaxSummaryViewSpec.scala:45]: No, but this has been put together to make the test work. Taking + " K" out leaves the failure: 

```nfo] - must contain details element with tax code information *** FAILED ***
[info]   [Definition of tax code part K.] not found in elements with '#employmentTaxCodeTerm_1_1' selector:[
[info]   	Definition of tax code part K. K] (IncomeTaxSummaryViewSpec.scala:43)
```

Manual viewing shows the text renders as intended "Definition of tax code part BR. " 
Note it is also designed to be hidden:

```<span class="govuk-visually-hidden">Definition of tax code part BR. </span>```

What are your thoughts? @sergi-hmrc

Comment (sergi-hmrc) [reply test/views/html/IncomeTaxSummaryViewSpec.scala:45]: ahhh okay!! i see

**Review by sergi-hmrc — APPROVED** [review]

**Review by masahmad — APPROVED** [review]

**Pull request merged**